protocol-definitions/Client.yaml

```diff
@@ -774,4 +774,112 @@ methods:
       retryable: true
       partitionIdentifier: -1
     response:
-      {}
\ No newline at end of file
+      {}
+  - id: 17
+    name: addMigrationListener
+    since: 2.2
+    doc: |
+      Adds a Migration listener to the cluster.
+    request:
+      retryable: false
+      partitionIdentifier: -1
+      params:
+        - name: localOnly
+          type: boolean
+          nullable: false
+          since: 2.2
+          doc: |
+            if true only node that has the migration sends the request, if false
```

Comment:
nit: consider changing to:
```
If set to true, the server adds the listener only to itself, otherwise the listener is is added for all members in the cluster.
```

Reply:
Sound better, thanks  👍